ext/mri/2.3.0/id_table.h

```diff
@@ -0,0 +1,30 @@
+#ifndef RUBY_ID_TABLE_H
+#define RUBY_ID_TABLE_H 1
+#include "ruby/ruby.h"
+
+struct rb_id_table;
```

Comment:
see also http://stackoverflow.com/questions/2700646/dereferencing-pointer-to-incomplete-type